test/testcafe/spec/DeviceSSOExtensionView_spec.js

```diff
@@ -157,16 +160,21 @@ test
   });
 
 test
-  .requestHooks(logger, verifyErrorMock)('hideds the spinner when authenticator verification fails', async t => {
+  .requestHooks(logger, verifyErrorMock)('hides the spinner when authenticator verification fails', async t => {
     const ssoExtensionPage = new BasePageObject(t);
     await ssoExtensionPage.navigateToPage();
-    await t.expect(Selector('.spinner').getStyleProperty('display')).eql('none');
+    await ssoExtensionPage.formExists();
+    await ssoExtensionPage.form.waitForErrorBox();
+    if(!userVariables.v3) {
```

Comment:
not necessary to check for spinner in v3 since it goes to a new page to show the error (terminal view).